roles/pulp_database_config/tasks/generate_database_fields_key.yml

```diff
@@ -1,4 +1,19 @@
 ---
+- name: Ensure python-cryptography is installed
+  package:
+    name: '{{ pulp_python_cryptography }}'
+    state: present
+  become: true
```

Comment:
I thought the purpose of doing this was to enable ansible modules like `openssl_privatekey` to work.

So I do not see how it is needed here.